pygrametl/__init__.py

```diff
@@ -444,23 +444,38 @@ def now(ignoredtargetconn=None, ignoredrow=None, ignorednamemapping=None):
 
 
 def ymdparser(ymdstr):
```

Comment:
Maybe remove `str` from `ymdstr` as the input need not be a string, e.g., `ymdornone`.

Reply:
That is a good point. It would, however, break backwards compatibility if anyone has used a named parameter.

Reply:
That is a better point, let us leave it.